src/backend/fold.js

```diff
@@ -42,6 +43,54 @@ function checkNullHtml(html) {
   return (html === '');
 }
 
+function getHoursFromTime(time) {
+  return time.split(':')[0];
+}
+
+function getMinutessFromTime(time) {
+  return time.split(':')[1];
+}
+
+function getTimeDifferenceOfDates(startTime, sessionTime) {
+  const firstDate = new Date(new Date().getFullYear(), new Date().getMonth(), new Date().getDate(), parseInt(getHoursFromTime(startTime)), parseInt(getMinutessFromTime(startTime)), 0);
+  const secondDate = new Date(new Date().getFullYear(), new Date().getMonth(), new Date().getDate(), parseInt(getHoursFromTime(sessionTime)), parseInt(getMinutessFromTime(sessionTime)), 0);
+  return (secondDate - firstDate);
+}
+
+function convertTimeToPixel(startTime, sessionTime) {
+  const timeDiff = getTimeDifferenceOfDates(startTime, sessionTime);
+
+  const top = timeDiff * timeToPixel/ (1000 * 60 * 15) + timeToPixel; // distance of session from top of the table
```

Comment:
@aayusharora That is exactly what I'm doing.

1. Get the start time of the first event and use it to calculate top for each session.
2. We are using `timeToPixel ` which is used to keep the pixel value corresponding to 15 mins.
3. We take time difference in `timeDiff` relative to the start time we saved earlier, and compute the top using formula `timeDiff * timeToPixel / (1000 * 60 * 15) + timeToPixel;`

Explanation : 

- `timeDiff` is difference in time in ms, we must convert it to minutes, we divide `timeDiff` by 1000 * 60 to convert it to minutes

- we add `timeToPixel` to the resulting value to make sure the the sessions start after the room heading. All room headings have `height = timeToPixel`